.travis.yml

```diff
@@ -28,6 +28,14 @@ matrix:
       after_success:
         - export COVR_RUNNING=true
         - Rscript -e 'covr::codecov()'
+    - language: java
+      dist: trusty
+      cache: packages
+      before_install:
+        - export MLFLOW_TEST_REQUIREMENTS=false
+      script:
+        - cd mlflow/java
+        - mvn clean package -q
```

Comment:
We do need to be within a Python conda env for the tests to pass

Reply:
Yep makes sense - AFAICT the top-level`install` section runs across the different languages, so we should be within a Python conda env with MLflow installed when this section is run